content/en/docs/Writing policies/mutate.md

```diff
@@ -351,6 +351,190 @@ The anchor processing behavior for mutate conditions is as follows:
 
 2. Next, all tag-values without anchors and all add anchor tags are processed to apply the mutation.
 
+## Mutate Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports mutate existing resources with `patchesStrategicMerge` and `patchesJson6902`. Unlike standard mutate policies that are applied through admission review, mutate existing policies are applied in the background which updates existing resources in the cluster. Such policies are triggered via admission requests, and can be optionally configured to apply on policy updates.
```

Comment:
"...Kyverno supports the mutation of existing resources..."
"...applied through the AdmissionReview process,..."

Last sentence suggestion:

"These mutate existing policies, like traditional mutate policies, are still triggered via the AdmissionReview process but apply to existing--and even different--resources. They may also optionally be configured to apply upon updates to the policy itself."